### ISSUE TITLE
Check indentation of multi-line commands in command subs

### ISSUE THREAD

**Issue by nmoroze** [nmoroze/tclint#23]
E.g. the following should flag a violation:

```tcl
set foo [cmd \
arg1]
```

**Comment by nmoroze** [comment]
Closing, indentation checks have been removed as of tclint v0.5.
